operations/research-and-development/product/product-planning/how-to-use-productboard.md

```diff
@@ -113,113 +114,113 @@ Two very important notes regarding this integration:
+You can add a new note to capture feedback from a customer call. This is used to help identify customer problems, wants/needs, feature shortcomings, and any other areas for improvement that stem from interacting with customers. Essentially, think of this as a place to communicate the most important takeaways from customer calls that the Product team can help you with.
 
 Important points about notes:
 
 * Notes are specific to a customer/prospect.
-* Notes can include multiple requests (PMs can apply multiple elements from the note to different features if necessary).
-* If you are not sure if the request or feedback should apply to an existing feature, you can simply capture the feedback as a note, assign it to the PM overseeing that area, or leave it unassigned.  
+* Notes can include multiple requests (Product Managerss can apply multiple elements from the note to different features if necessary).
+* If you are not sure if the request or feedback should apply to an existing feature, you can simply capture the feedback as a note, assign it to the Product Manager overseeing that area, or leave it unassigned.
 
-### 2d - Adding insights directly to an existing feature
+### 2d Adding insights directly to an existing feature
 
-As a **Contributor** in ProductBoard, you have access to view the [master feature list](https://mattermost.productboard.com/feature-board/1097524-master-feature-list). This is the view where the Product team organizes and prioritizes new features and functionality that is being considered or planned for Mattermost. 
+As a **Contributor** in productboard, you have access to view the [Master Feature list](https://mattermost.productboard.com/feature-board/1097524-master-feature-list). This is the view where the Product team organizes and prioritizes new features and functionality that is being considered or planned for Mattermost.
 
-You can search and browse all features and ideas and even read the notes and portal cards from PMs for a detailed description of the feature/idea. In the case of planned features, you will have complete insight into the status of the feature and a rough idea of where it sits on the roadmap. 
+You can search and browse all features and ideas and even read the notes and portal cards from Product Managers for a detailed description of the feature/idea. In the case of planned features, you'll have complete insight into the status of the feature and a rough idea of where it sits on the roadmap.
 
-Within the detailed view of individual features, you can view the full list of insights added by other PMs, CSMs, SAs, or Support Managers. You also have the option to add insight directly to the feature itself.
+Within the detailed view of individual features, you can view the full list of insights added by other Product Managers, Customer Success Managers, Solutions Architects, or Support Managers. You also have the option to add insight directly to the feature itself.
```

Comment:
Confirming that these internal roles are accurate or whether they need to be updated/reviewed.

Reply:
Only changes are:
`Solution Architects` => `Customer Engineers` 
`Support Managers` => `Support Engineers`